interface/modules/custom_modules/oe-module-comlink-telehealth/sql/table.sql

```diff
@@ -0,0 +1,88 @@
+#EndIf
+
+#IfNotRow openemr_postcalendar_categories pc_constant_id comlink_telehealth_new_patient
+INSERT INTO `openemr_postcalendar_categories` (
+    `pc_constant_id`, `pc_catname`, `pc_catcolor`, `pc_catdesc`,
+   `pc_recurrtype`, `pc_enddate`, `pc_recurrspec`, `pc_recurrfreq`, `pc_duration`,
+   `pc_end_date_flag`, `pc_end_date_type`, `pc_end_date_freq`, `pc_end_all_day`,
+   `pc_dailylimit`, `pc_cattype`, `pc_active`, `pc_seq`, `aco_spec`
+)
+VALUES (
+    'comlink_telehealth_new_patient', 'Telehealth New Patient', '#a2d9e2'
+    , 'New Patient Telehealth appointments', '0', NULL
+    , 'a:5:{s:17:"event_repeat_freq";s:1:"0";s:22:"event_repeat_freq_type";s:1:"0";s:19:"event_repeat_on_num";s:1:"1";s:19:"event_repeat_on_day";s:1:"0";s:20:"event_repeat_on_freq";s:1:"0";}'
+    , '0', '1800', '0', NULL, '0', '0', '0', 0, '1', '10', 'encounters|notes'
+);
+#EndIf
+
+#IfNotRow openemr_postcalendar_categories pc_constant_id comlink_telehealth_established_patient
+INSERT INTO `openemr_postcalendar_categories` (
+    `pc_constant_id`, `pc_catname`, `pc_catcolor`, `pc_catdesc`,
+    `pc_recurrtype`, `pc_enddate`, `pc_recurrspec`, `pc_recurrfreq`, `pc_duration`,
+    `pc_end_date_flag`, `pc_end_date_type`, `pc_end_date_freq`, `pc_end_all_day`,
+    `pc_dailylimit`, `pc_cattype`, `pc_active`, `pc_seq`, `aco_spec`
+)
+VALUES (
+   'comlink_telehealth_established_patient', 'TeleHealth Established Patient', '#93d3a2'
+    , 'TeleHealth Established Patient appointment', '0', NULL
+    , 'a:5:{s:17:"event_repeat_freq";s:1:"0";s:22:"event_repeat_freq_type";s:1:"0";s:19:"event_repeat_on_num";s:1:"1";s:19:"event_repeat_on_day";s:1:"0";s:20:"event_repeat_on_freq";s:1:"0";}'
+    , '0', '900', '0', NULL, '0', '0', '0', 0, '1', '9', 'encounters|notes'
+);
+#EndIf
+
+#IfNotTable comlink_telehealth_appointment_session
+CREATE TABLE IF NOT EXISTS `comlink_telehealth_appointment_session`
+(
+    `id`              	        BIGINT(20)         NOT NULL AUTO_INCREMENT,
+    `user_id`                   BIGINT(20)         NOT NULL COMMENT 'Foreign key reference to users.id',
+    `pc_eid`                    INT(11) UNSIGNED   NOT NULL COMMENT 'Foreign key reference to openemr_postcalendar_events.pc_eid',
+    `encounter`                 BIGINT(20)         NOT NULL COMMENT 'Foreign key reference to forms.encounter',
+    `pid`                       BIGINT(20)         NOT NULL COMMENT 'Foreign key reference to patient_data.pid',
+    `provider_start_time`       DATETIME           DEFAULT NULL DEFAULT CURRENT_TIMESTAMP  COMMENT 'Provider start time',
+    `provider_last_update`      DATETIME           DEFAULT NULL COMMENT 'Provider last communication timestamp',
+    `patient_start_time`        DATETIME           DEFAULT NULL COMMENT 'Patient join time',
+    `patient_last_update`       DATETIME           DEFAULT NULL COMMENT 'Last communication timestamp with patient',
+    PRIMARY KEY (`id`)
+) ENGINE = InnoDB;
+#EndIf
+
+#IfNotRow2D apptstat option_id TRNSFR
```

Comment:
```
#IfNotRow2D list_options list_id apptstat option_id TRNSFR
```

Reply:
Changed this up, tested with a fresh install and its working for me now.